### ISSUE TITLE
Can it be set as 100% based on Seconds?

### ISSUE THREAD

**Issue by Krammig** [kottenator/jquery-circle-progress#210]
Is it possible to set say  20 Seconds and then have the progress move through 100% travel over that specified time interval ?

**Comment by kottenator** [comment]
Yes, of course - you can use `setTimeout(() => {/* start animation */}, 20000)`.
